SPEC/FILES.md

```diff
@@ -58,13 +60,64 @@ ipfs.files.add(files, function (err, files) {
 
 A great source of [examples][] can be found in the tests for this API.
 
-#### `createAddStream`
+#### `AddReadableStream`
 
 > Add files and data to IPFS using a transform stream.
 
 ##### `Go` **WIP**
 
-##### `JavaScript` - ipfs.files.createAddStream([options], [callback])
+##### `JavaScript` - ipfs.files.AddReadableStream([options], [callback])
```

Comment:
please don't capitalize method names in JavaScript, that makes for some very surprising apis.

Reply:
ups, typo